src/containers/EventContainer/EventContainer.js

```diff
@@ -3,34 +3,50 @@ import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
 import EventCard from '../../components/EventCard/EventCard';
 
-export const EventContainer = ({ pathname, events }) => {
+export const EventContainer = ({ pathname, events, searchText }) => {
   const styles = pathname.includes('profile') ? 'EventContainer profile' : 'EventContainer home';
-    return (
-      <div className={styles}>
+
+  const filterEvents = () => {
+    return events.filter(event => {
+      const { name, city, state } = event.attributes;
+      const text = name + ' ' + city + ' ' + state;
+      return text.toUpperCase().includes(searchText.toUpperCase())
```

Comment:
Nice job remembering toUpperCase() both of these - has totally caused me silly errors in the past! 🤣